pkg/util/tracing/tracingpb/recording.go

```diff
@@ -73,13 +72,13 @@ type traceLogData struct {
 //
 // TODO(andrei): this should be unified with
 // SessionTracing.generateSessionTraceVTable().
-func (r Recording) String() string {
-	if len(r) == 0 {
+func RecordingToString(r *Recording) string {
```

Comment:
(something I learned recently that you might find useful) PS: it's possible to define `(gogoproto.goproto_stringer) = false;` on proto types to override .String() with your own.

Reply:
ahhh, I googled for exactly this yesterday and came back empty-handed. Good to know!